packages/babel-plugin-optimize-react/__tests__/hooks-test.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+const plugin = require('../index.js');
+const babel = require('@babel/core');
+
+function transform(code) {
+  return babel.transform(code, {
```

Comment:
this is gonna load babelrc / babel.config.js, not sure if you want that here

also - would be good to have tests both with & without `@babel/plugin-transform-destructuring` (and possibly some other overlapping combinations)

Reply:
I see you have no babelrc files in this package, but probably better disable config loading with
```js
babel.transform(code, {
  babelrc: false,
  configFile: false,
  plugins: [plugin],
}).code
```
just to be more future proof

Reply:
Good call on testing with other plugins. It seems to be ignored when used alongside `preset-env`: [optimize-react + preset-env](https://babeljs.io/en/repl#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=JYWwDg9gTgLgBAJQKYEMDGMA0cDeBXAZyQGUYUYkBfOAMyghDgCIpUMmBuAKDl76QAekWLTwA7DMAhi4AWQCeAYQaQxSMTAAUASlw8-BtNILwA2kfEUo2IjGWWkUALpwAvHEIkyFTQAZt3AYGrDB4UDLI6DAAdGis5EgAogA2SCDqWkwAJsAAbkzYYnjJydgWGo4B-gaUQA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=react%2Cenv&prettier=false&targets=&version=7.5.5&externalPlugins=babel-plugin-optimize-react%400.0.4).

I'll work on a PR to this one. At least with a test, hopefully with a fix.